### PR TITLE
Subdomain requirement update

### PR DESCRIPTION
**Pull request by albertchen-okta**
proposed some new language to clarify the subdomain suggestion

## Description:
- We do not have any validation or restriction against a user using a root domain or non-subdomain for custom domains, so editing this guide to reflect that.

### Resolves:

* https://oktainc.atlassian.net/browse/OKTA-412340

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/overview/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Okta serves pages on your custom domain over HTTPS. To set up this feature, you
 
 ### Caveats
 
-* Your custom domain must be a [subdomain](https://en.wikipedia.org/wiki/Subdomain).
+* Most custom domains are configured using one of your [subdomains](https://en.wikipedia.org/wiki/Subdomain). However, there are no restrictions on using any other domain such as your root domain, if that fits your use-case for your sign-in page taking over your main website.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-url-domain/overview/index.md:17]:
```suggestion
* Most custom domains are configured using one of your [subdomains](https://en.wikipedia.org/wiki/Subdomain). However, there are no restrictions on using any other domain such as your root domain, if that fits your use case for your sign-in page taking over your main website.
```

**Review by susanharper-okta — COMMENTED** [review]
I think root makes sense. LGTM with one minor nit.

**Review by anitadongare-okta — APPROVED** [review]
lgtm 🚢

**Review by alexdahl-okta — APPROVED** [review]
Thank you!

**Pull request merged**